public/app/plugins/panel/canvas/editor/elementEditor.tsx

```diff
@@ -57,18 +63,21 @@ export function getElementEditor(opts: CanvasEditorOptions): NestedPanelOptions<
     // Dynamically fill the selected element
     build: (builder, context) => {
       const { options } = opts.element;
-      const layerTypes = canvasElementRegistry.selectOptions(
-        options?.type // the selected value
-          ? [options.type] // as an array
-          : [DEFAULT_CANVAS_ELEMENT_CONFIG.type]
-      );
+      const current = options?.type ? [options.type] : [DEFAULT_CANVAS_ELEMENT_CONFIG.type];
+      const layerTypes = getElementTypes(opts.scene.shouldShowAdvancedTypes, current);
+
+      const isUnsupported =
+        !opts.scene.shouldShowAdvancedTypes && !defaultElementItems.filter((item) => item.id === options?.type).length;
 
       builder.addSelect({
         path: 'type',
         name: undefined as any, // required, but hide space
         settings: {
-          options: layerTypes.options,
+          options: layerTypes,
         },
+        description: isUnsupported
+          ? 'Selected element type is not supported by current settings. Please enable advanced element types.'
+          : '',
```

Comment:
I think it might make sense to but the `Layout` section right underneath the element section (above background / border) as I think the layout would be used more so than the current two sections above it
![Screen Shot 2022-07-29 at 12 13 50 PM](https://user-images.githubusercontent.com/22381771/181828998-894e5e90-21e4-4871-a02f-52bed41d9141.png)